frontend/packages/kubevirt-plugin/src/utils/validations/template/template-validations.ts

```diff
@@ -63,50 +70,67 @@ export class TemplateValidations {
     if (this === otherTempValidations) {
       return true;
     }
+    const isBusesEqual = (buses, otherBuses) =>
```

Comment:
+1 for having a function. Can we move it one layer up to the class and make it private?